packages/validation/src/loadbalancers.schema.ts

```diff
@@ -50,10 +50,14 @@ export const EndpointSchema = object({
 
 const HealthCheckSchema = object({
   protocol: string().oneOf(['http', 'tcp']),
-  interval: number().min(0),
-  timeout: number().min(0),
-  unhealthy_threshold: number().min(0),
-  healthy_threshold: number().min(0),
+  interval: number().typeError('Inteval must be a number.').min(0),
+  timeout: number().typeError('Timeout must be a number.').min(0),
+  unhealthy_threshold: number()
+    .typeError('Unhealthy Threshold must be a number.')
+    .min(0),
+  healthy_threshold: number()
+    .typeError('Healthy Threshold must be a number.')
+    .min(0),
```

Comment:
```suggestion
  interval: number().typeError('Interval must be a number.').min(1, 'Interval must be greater than zero.'),
  timeout: number().typeError('Timeout must be a number.').min(1, 'Timeout must be greater than zero.'),
  unhealthy_threshold: number()
    .typeError('Unhealthy Threshold must be a number.')
    .min(1, 'Unhealthy Threshold must be greater than zero.'),
  healthy_threshold: number()
    .typeError('Healthy Threshold must be a number.')
    .min(1, 'Healthy Threshold must be greater than zero.'),
```

According to the tooltips, we need min values of 1 for these fields. Also adding a suggestion to better handle a validation message if a user inputs a number below the minimum. Current behavior:

![Screenshot 2023-12-11 at 2 10 38 PM](https://github.com/linode/manager/assets/114685994/8a20f99a-23c8-41a0-916b-22f5f16e6d8d)

Reply:
Great suggestions! Thank you!